src/types/artist.ts

```diff
@@ -4,6 +4,11 @@ import { Record, TimeField } from './record'
 
 export type Artist = Record & {
   name: string;
+  address?: string;
+  avatarUrl?: string;
+  externalLinks: any[]; // TODO: build out a TS type with name: string, type: string, link: string
+  theme: any[]; // TODO: check for an existing TS type or define one
+  spinampLayoutConfig: any[]; // TODO build out a TS type for type: string, data: string, visible: boolean
```

Comment:
maybe let's leave it dynamic for now as just any (not any[]) since i think it may change a bunch in time?

Reply:
also can be optional?